enginetest/queries/priv_auth_queries.go

```diff
@@ -341,9 +341,9 @@ var UserPrivTests = []UserPrivilegeTest{
 						uint16(2),               // Trigger_priv
 						uint16(2),               // Create_tablespace_priv
 						uint16(1),               // ssl_type
-						"",                      // ssl_cipher
-						"",                      // x509_issuer
-						"",                      // x509_subject
+						[]byte(""),              // ssl_cipher
```

Comment:
Some places you're using uint8 and others byte. Standardize on byte everywhere you're doing this